src/templates/admin-settings-Spectra/admin-settings-Spectra.stories.js

```diff
@@ -0,0 +1,510 @@
+
+							{ /* Setting Section */ }
+							<Container className="mb-0.5" align="center">
+								<Container.Item className="w-full max-w-120 space-y-1">
+									<Label size="md" className="font-semibold">
+										Container Elements Gap
+									</Label>
+									<Label
+										tag="p"
+										size="sm"
+										variant="help"
+										className="m-0"
+									>
+										This setting will apply default Row &
+										Column Gaps in the Container Block.
+									</Label>
+								</Container.Item>
+								<Input
+									defaultValue={ 20 }
+									suffix={
+										<span className="text-badge-color-gray rounded border border-solid border-border-subtle p-0.5 text-center text-xs font-medium bg-badge-background-gray">
+											PX
+										</span>
+									}
+									type="number"
+								/>
+							</Container>
+
+							<hr className="w-full border-b-0 border-x-0 border-t border-solid border-t-border-subtle" />
+
+							{ /* Setting Section */ }
+							<Container className="mb-0.5" align="center">
+								<Container.Item className="w-full max-w-95 space-y-1">
+									<div className="space-x-1 flex items-center">
+										<Label
+											className="font-semibold"
+											size="md"
+										>
+											Dynamic Content
+										</Label>
+										<span className="inline-block py-0.5 px-1 text-text-inverse bg-background-inverse border border-solid border-border-inverse text-xs font-medium rounded-full">
+											PRO
+										</span>
+									</div>
+									<Label
+										tag="p"
+										size="sm"
+										variant="help"
+										className="m-0"
+									>
+										Choose how you want to display dynamic
+										content settings.
+									</Label>
+								</Container.Item>
+								<Container.Item className="ml-auto">
+									<RadioButton.Group
+										className="[&_button]:text-xs [&_button]:font-semibold"
+										size="sm"
+										columns={ 2 }
+										style="tile"
+										disableGroup
+									>
+										<RadioButton.Button value="val1">
+											Popup
+										</RadioButton.Button>
+										<RadioButton.Button value="val2">
+											Sidebar
+										</RadioButton.Button>
+									</RadioButton.Group>
+								</Container.Item>
+							</Container>
+
+							<div className="w-full flex items-center justify-between border border-solid border-brand-200 bg-brand-background-50 py-2 pl-2.5 pr-2 rounded-lg">
+								<p className="m-0 text-sm text-text-primary font-normal">
+									<span className="font-semibold">
+										Personalized content
+									</span>{ ' ' }
+									delivers relevant content for higher
+									engagement.
+								</p>
+								<button className="inline-flex items-center p-0 m-0 bg-transparent outline-none focus:outline-none border-0 shadow-none space-x-1 text-xs font-semibold text-link-primary">
+									Upgrade now{ ' ' }
+									<ArrowUpRight className="size-3.5" />
+								</button>
+							</div>
```

Comment:
can we use force-ui Container here?

Reply:
Also we can use Button component with Icon

Reply:
As we discussed previously in the scrum call, we will create a new component for this. That's why used `div` for this section. Therefore marking this as resolved.